src/EventStore.Core/TransactionLog/LogRecords/LogRecord.cs

```diff
@@ -23,6 +23,10 @@ public abstract class LogRecord : ILogRecord {
 		}
 
 		public static ILogRecord ReadFrom(BinaryReader reader) {
+			return ReadFrom(reader, 0);
```

Comment:
should the length be equal to the binary reader's stream length here?

Reply:
removed this method